mechanical_turk/get_sent.py

```diff
@@ -0,0 +1,82 @@
+import os
```

Comment:
@martinbolger , @bshpringer should we move this file into the data_processing\\unlabeled_data folder?